### DISCUSSION TITLE
Google Fonts wont apply

### DISCUSSION THREAD

**Discussion by Morbe17** [vercel/community#517] [Help] [answered]
Hey guys, i recently published my app through Vercel and i'm having a small issue. I'm importing google fonts into  the .css with the link but they don't seem to apply.

If i run my app in localhost the fonts do apply properly but they dont through vercel.

@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');

**Comment by swarnava** [comment, marked as answer]
Hey, are you using Next.js?

**Reply by Morbe17** [reply]
Yes, i'm currently using NextJs

**Reply by swarnava** [reply]
Please Check [this](https://nextjs.org/docs/basic-features/font-optimization), and use `<style data-href`
